restler/unit_tests/log_baseline_test_files/Bug_Buckets_Json/InvalidValueChecker_500_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": "500",
+    "checkerName": "InvalidValueChecker",
+    "reproducible": true,
+    "verb": "PUT",
+    "endpoint": "/item/{itemName}",
+    "status_text": "Internal Server Error",
+    "request_sequence": [
+        {
+            "requestOrder": 0,
+            "replayrequest": "PUT /item/itemName<test!>-bc8aff2b56?location=centralus&date=2021-01-01 HTTP/1.1\r\nAccept: application/json\r\nHost: unittest\r\nContent-Type: application/json\r\nAUTHORIZATION TOKEN\r\n\r\n{\"datetest\":\"2021-01-01\",\"id\":\"/testparts/{BT\t[\"}\r\n",
+            "producer_timing_delay": 0,
+            "max_async_wait_time": 20,
+            "response": "HTTP/1.1 500 Internal Server Error\r\nRestler Test\r\n\r\n{\"code\": \"InternalServerError\", \"message\": \"Invalid control character at: line 1 column 46 (char 45)\"}"
```

Comment:
>response [](http://example.com/codeflow?start=13&length=8)

Could you please put the request right above the response?  This makes it easier to read.  I think putting them either at the very top or at the bottom of the object would be the most readable

Reply:
Updated